prow/gcsupload/run.go

```diff
@@ -136,9 +143,10 @@ func (o Options) assembleTargets(spec *downwardapi.JobSpec, extra map[string]gcs
 }
 
 // PathsForJob determines the following for a job:
-//  - path in blob storage under the bucket where job artifacts will be uploaded for:
-//     - the job
-//     - this specific run of the job (if any subdir is present)
+//   - path in blob storage under the bucket where job artifacts will be uploaded for:
+//   - the job
+//   - this specific run of the job (if any subdir is present)
+//
```

Comment:
I think your formatter has changed this. We should leave it as is to reduce merge conflicts